cfme/tests/configure/test_default_views_cloud.py

```diff
@@ -21,6 +22,16 @@
 
 # TODO refactor for setup_provider parametrization with new 'latest' tag
 
+def test_default_view_cloud_reset(appliance, soft_assert):
+    """
+        This test case performs Reset button test.
+    """
+    navigate_to(appliance.user.my_settings, "DefaultViews")
+    view = appliance.browser.create_view(DefaultViewsForm)
```

Comment:
**REQUIRED**
`navigate_to(appliance.user.my_settings, "DefaultViews")` returns an instance of `DefaultViewsForm`